docker/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM python:3.7
+
+# Better movement with [Ctrl + →] and [Ctrl + ←]
+ENV INPUTRC=/etc/inputrc
+
+RUN set -ex \
+  && echo '"\e[1;5C": forward-word' >> /etc/inputrc \
+  && echo '"\e[1;5D": backward-word' >> /etc/inputrc
+
+# Set working directory.
+# Mount your project under the same path.
+WORKDIR /usr/src
+
+# Install useful programs
+RUN apt-get update -y && apt-get install -y \
+      less \
+      git
+
+# Install DVC, supporting all remotes
+RUN pip install "dvc[all]"
+
+# Enable autocompletion scripts for DVC
+RUN wget \
+    -O /etc/bash_completion.d/dvc \
+    https://raw.githubusercontent.com/iterative/dvc/master/scripts/completion/dvc.bash
+
+# Add a user to run DVC
+ARG USER_ID=1000
```

Comment:
I suppose you are doing this so that files created by dvc are not owned by root, right? If so, now they are going to be owned by UID 1000, which is absolutely not guaranteed to match your current user on the host.

Reply:
The user must build the image with their UID, see the description of the issue:
https://github.com/iterative/dvc/pull/2844#issue-345476189